presto-docs/src/main/sphinx/release/release-0.103.rst

```diff
@@ -7,3 +7,28 @@ Hive Changes
 ------------
 
 * Removed the ``hive.max-split-iterator-threads`` parameter and renamed ``hive.max-global-split-iterator-threads`` to ``hive.max-split-iterator-threads``.
+
+Task Parallelism
+----------------
+Queries involving a large number of aggregations or a large hash table for a
+join can be slow due to single threaded execution in the intermediate stages.
+This release adds experimental configuration and session properties to execute
+this single threaded work in parallel.  Depending on the exact query this may
+reduce wall time, but will likely increase CPU usage.
+
+Use the configuration parameter ``task.default-concurrency`` or the session
+property ``task_default_concurrency`` to set the default number of parallel
+workers to use for join probe, build and final aggregation.  Additionally,
+the session properties ``task_join_concurrency``, ``task_hash_build_concurrency``
+and ``task_aggregation_concurrency`` can be used to control the parallelism
+for each type of work.
+
+This is an experimental feature and will likely change in a future release.  It
+also expected that this will eventually be handled automatically by the query
+planner and these options will be removed entirely.
+
+General Changes
+---------------
+* Change http client default idle timeout to 2 second
```

Comment:
Make all these one sentence:
    * Change HTTP client defaults: 2 second idle timeout, 10 second request timeout and 250 connections per host.